src/components/analytics/__tests__/PatternVisualizationReact.test.tsx

```
@@ -76,19 +77,24 @@ vi.mock('recharts', () => {
     ResponsiveContainer: ({ children }: { children: React.ReactNode }) => (
       <div data-testid="responsive-container">{children}</div>
     ),
+
     LineChart: ({ children }: { children: React.ReactNode }) => (
```

ESLint

[error] 'React' is not defined.

(no-undef)